Data-Science/MLflow/bike-sharing-mlflow.ipynb

```diff
@@ -31,10 +38,12 @@
    "source": [
     "import os\n",
     "\n",
-    "# Configure s3 details\n",
     "os.environ[\"AWS_ACCESS_KEY_ID\"] = \"s3\"\n",
     "os.environ[\"AWS_SECRET_ACCESS_KEY\"] = os.environ['MLFLOW_TRACKING_TOKEN']\n",
```

Comment:
```suggestion
    "os.environ[\"AWS_ACCESS_KEY_ID\"] = os.environ['MLFLOW_TRACKING_TOKEN']\n",
    "os.environ[\"AWS_SECRET_ACCESS_KEY\"] = \"s3\"\n",
```

Reply:
Thank you for this , I had forgotten to revert it to when testing. I've pushed a commit to correct this now.